README.md

```diff
@@ -139,3 +139,124 @@ if you don´t have the tools for fuzzing installed use
 make fuzzer_tools
 ```
 
+## Benchmarks
+
+To get the results of the table below, run
+
+```
+make benchmarks_table
+```
+
+The results shown are from the execution of a Fibonacci program.
+
+First table has the results that are independent of the hardware used.
+
+| n   | Trace length | Trace time | Proof size 80 | Proof size 128 |
+|-----|--------------|------------|---------------|----------------|
+| 100 | 2^10         | 0.9 ms     | 270 KB        | 1.2 MB         |
+| 500 | 2^12         | 5.3 ms     | 335 KB        | 1.5 MB         |
+| 2k  | 2^14         | 24.7 ms    | 407 KB        | 1.8 MB         |
+| 5k  | 2^16         | 77.2 ms    | 488 KB        | 2.2 MB         |
+| 20k | 2^18         | 312 ms     | 576 KB        | 2.6 MB         |
+
```

Comment:
It's not clear 80 and 128 are security bits, we should add that to the text

We should make one of conjecturable, and another one of provable. This is mixing both and is confusing

Reply:
Since others projects are benchmarking with conjecturable, we should do that too, the other one is not necessary for this public data